src/aiod/authorisation/authorisation.py

```diff
@@ -0,0 +1,80 @@
+import os
+from keycloak import KeycloakOpenID
+
+from aiod.config.settings import server_url, client_id, realm
+
+keycloak_openid = KeycloakOpenID(
+    server_url=server_url,
+    client_id=client_id,
+    realm_name=realm,
+)
+
+
+def login(username: str, password: str) -> None:
+    """
+    Logs in the user with the provided username and password.
+
+    Args:
+        username (str): The username of the user.
+        password (str): The password of the user.
+
+    Raises:
+        MissingCredentialsError: If the username or password is missing.
+        MissingTokenError: If Keycloak returns an invalid token.
+    """
+    if username is None or password is None:
+        raise MissingCredentialsError(
+            "Username and/or password missing! Please provide your credentials and try again."
+        )
+    token = keycloak_openid.token(username, password)
+
+    if not token:
+        raise MissingTokenError("Keycloak returned an invalid token!")
+    os.environ["ACCESS_TOKEN"] = token["access_token"]
+    os.environ["REFRESH_TOKEN"] = token["refresh_token"]
```

Comment:
Why are we storing it here? I realize that it stays local to the Python process, but do we sometimes expect the user to "warm-start" their script with these variables already preloaded in their environment (and thus not needing to call `login` in the first place)?

Reply:
Yes, you're totally right. Everything can go to some sort of config object.